README.md

```diff
@@ -219,6 +219,14 @@ spec:
 It is notable that this is not the default (anymore) because certain popular
 registries (including Docker Hub) do not support multi-level repository names.
 
+`ko resolve`, `ko apply`, and `ko create` accept an optional `--selector` or `-l` 
```

Comment:
Part of me wants to move this down to the "release management" section, because that's where I see this being most useful... though it could always be useful during development. WDYT?

Reply:
my thinking was that it's useful for any `ko` user who wants to run these commands... they might not be checking the "release management" section (as in my case, i've been using it as part of the serving e2e tests)